pkg/controllers/management/k3sbaseupgrade/deployPlans.go

```diff
@@ -59,13 +79,12 @@ func (h *handler) deployPlans(cluster *v3.Cluster) error {
 
 		} else {
 
-			switch name := plan.Name; name {
-			case k3sMasterPlanName:
+			if name := plan.Name; strings.Contains(name, "master-plan") {
```

Comment:
Keep case switch and just add new case: 
`case k3sMasterPlanName, rke2MasterPlanName:`
Less brittle that way, if someone adds a third and doesn't want to use this "master-plan" sub-string.